src/hooks/useArNSReturnedName.tsx

```diff
@@ -0,0 +1,16 @@
+import { useGlobalState } from '@src/state';
+import { useQuery } from '@tanstack/react-query';
+
+export function useArNSReturnedName({ name }: { name: string }) {
+  const [{ arioContract }] = useGlobalState();
+
+  return useQuery({
+    queryKey: ['arns-returned-name', name],
+    queryFn: () => {
+      if (!name) return undefined;
+      return arioContract.getArNSReturnedName({ name }).catch(() => undefined);
+    },
+    enabled: !!name,
+    staleTime: 24 * 60 * 60 * 1000, // 24 hours
+  });
+}
```

Comment:
see other comment on name length, null, and throwing patterns

Reply:
there was an existing hook, so updated it [c5fc1a7](https://github.com/ar-io/arns-react/pull/854/commits/c5fc1a729c63670f4cb744f046b78cbddbb722bc)